lib/OnboardingMessageProvider.jsm

```diff
@@ -9,10 +9,13 @@ ChromeUtils.import("resource://gre/modules/addons/AddonRepository.jsm");
 
 async function getAddonName() {
   try {
-    const {content} = await AttributionCode.getAttrDataAsync();
+    let {content} = await AttributionCode.getAttrDataAsync();
     if (!content) {
       return null;
     }
+    while (content.includes("%")) {
+      content = decodeURIComponent(content);
```

Comment:
An actual `%` without an invalid encoding will result in `URIError: malformed URI sequence` e.g.,

`decodeURIComponent("%")`

Reply:
I was just working on fixing that. 😁